set.mm

```diff
@@ -64215,6 +64217,14 @@ empty set when it is not meaningful (as shown by ~ ndmfv and ~ fvprc ).
       HUJCKZABLZUFUGUMCAEBDNOULUOECUNCPHZABUNUJPHUPUIDQUJCPRSUAUHCKZUKUNABUQUKC
       UJKUNUHCUJUBCUJUCTUDUET $.
 
+    $( A member of a surjective function's ran is a value of the function.
```

Comment:
To me, the range is the set of values of a function, so this comment seems a bit tautological. If F is a function from A to B, I would call B the codomain. Of course this is not definable in the current set.mm because of the way functions are defined. This does not forbid using the term "codomain" in comments.

Reply:
I agree, "surjective function's ran" should be changed to "surjective function's  codomain"

Reply:
Thanks! I've changed to "codomain".